tests/storage/cdi_upload/conftest.py

```diff
@@ -91,3 +92,10 @@ def uploaded_dv_scope_class(unprivileged_client, namespace, storage_class_name_s
         ) as upload_result:
             check_upload_virtctl_result(result=upload_result)
             yield dv
+
+
+@pytest.fixture()
+def storage_profile_minimum_supported_pvc_size(storage_class_name_scope_function):
+    storage_profile = StorageProfile(name=storage_class_name_scope_function)
+    annotations = getattr(storage_profile.instance.metadata, "annotations", {}) or {}
+    return annotations.get("cdi.kubevirt.io/minimumSupportedPvcSize", "1Gi")
```

Comment:
Please do not use the hard-coded "1Gi". 
I think it's better to return `None` if there's no annotation.
So when you use it, it'll be something like
`size=storage_profile_minimum_supported_pvc_size or DEFAULT_DV_SIZE`